### ISSUE TITLE
galaxyline plugin may cause significant perf issue.

### ISSUE THREAD

**Issue by towry** [Avimitin/nvim#73]
My fork: https://github.com/pze/nvim

The file to test: https://gist.githubusercontent.com/towry/a6820e5361c262df7807adfccf772b07/raw/8a5bb8b24df844420afba4dd0145fd1efb0e8763/logseq-theme.css

save it as a `custom.css` file, use nvim to edit it.

The issue:

By using my fork, the perf is very bad, the cursor move can't follow my keystroke, the notify ui fading effect even stuck.

I tried using upstream/master branch, the issue seems gone, so i figured it maybe my modification issue? I tear down the modification one by one, no luck.

So i use `stevearc/profile.nvim` to make a test, the result file is here: https://gist.githubusercontent.com/towry/1030ab651ddec2af5391b6e03ff15080/raw/34ee49238e412273f3cfe42341cce8230a9385f7/galaxyline_profiling.json

From the result, it is the galaxyline plugin taken too much time, so disabled it, THE ISSUE GONE.

I think it is worth mention this here even it maybe is my modification's fault.

**Comment by Avimitin** [comment]
Do you enable Treesitter for CSS?

**Comment by towry** [comment]
Yes, with css TS enabled, the perf is better, I think this is the root cause.

Thanks.
